README.md

```diff
@@ -4,6 +4,12 @@ The Student Robotics public documentation.
 
 ## Getting Started
 
+For ease of setup, a Docker container is provided. Simply install Docker and `docker-compose`, and run `docker-compose up`.
+
+Once setup, the site will be accessible on http://localhost:4000/docs/
```

Comment:
This section makes no mention of how the user should:
a) update the build and running images when they make changes to the local files they're editing
b) run the validation logic (spell checking, validation of various data files, etc.)

Admittedly the README as it stands is also missing instructions on running `rake validate`, however given the relative simplicity (and tab-completed-ness) of doing so that would be easy to fix. Without an easily accessible development setup, the docker approach would make running that considerably harder.

Reply:
> update the build and running images when they make changes to the local files they're editing

That's because there are no steps. Files are bind-mounted into the running container, so changes on the host are immediately reflected inside the container, and `jekyll` will rebuild them. The process for dependency updates is to rebuild the container, which yes isn't documented.

Reply:
Ah, I'd missed that it was also bound. I was looking at the `COPY` in the `Dockerfile`. Maybe that should be removed in favour of a comment about the `docker-compose.yml`.